core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -137,14 +137,8 @@ public void update(float delta) {
      * @param batch the batch used to draw with
      */
     public void draw(SpriteBatch batch) {
-        //batch.draw(cannon_texture, hitbox.x, hitbox.y, BALL_RAD, BALL_RAD);
-
-        batch.draw(cannonTextureRegion, xpos - width / 2f - 3, ypos - height / 2f + 23, 
-                width / 2f, height / 2f, width, height, 5 / 2, 1, angle, true);
-
-        //batch.draw(cannon_texture_region, x - width / 2f, y - height / 2f,    
-        //width / 2f, height / 2f, width, height, 1, 1, angle, true);
-
+        batch.draw(cannonTextureRegion, xpos - width / 2f + 11, ypos - height / 2f + 7, 
```

Comment:
They are just some small and different offsets to put the cannon in a good location and re-locate the rotation point to a place different than the cannon center (since it should rotate around a specific rotation point that's not exactly the center at height/2)
And nop they can't be eliminated by using height and width